test/configparser.cpp

```diff
@@ -44,6 +50,14 @@ TEST_CASE("evaluate_backticks replaces command in backticks with its output",
 				"a single literal backtick: \\`") ==
 			"a single literal backtick: `");
 	}
+	SECTION("commands with space are evaluated by backticks")
+	{
+		ConfigParser cfgparser;
+		KeyMap keys(KM_NEWSBOAT);
+		cfgparser.register_handler("bind-key", &keys);
+		REQUIRE_NOTHROW(cfgparser.parse("data/config-space-backticks"));
+		REQUIRE(keys.get_operation("s", "all"));
```

Comment:
Implicit casting confused me for a moment here: this reads like `get_operation()` returns a `bool`, and this doesn't align with the method name. I suggest being explicit: `REQUIRE_FALSE(keys.get_operation("s", "all") == OP_NIL)`.